build.gradle

```diff
@@ -14,11 +14,11 @@ gradlePlugin {
 }
 
 group 'org.hyperskill'
-version '1.8.3'
+version '2.0.0'
 
 java {
-    sourceCompatibility = JavaVersion.VERSION_17
-    targetCompatibility = JavaVersion.VERSION_17
+    sourceCompatibility = JavaVersion.VERSION_24
+    targetCompatibility = JavaVersion.VERSION_24
```

Comment:
Does this mean that users now need to install Java 24 separately to solve problems? After all, only Java 21 will come with the IDE